source/project-stories/plymouth-university.erb

```diff
@@ -0,0 +1,328 @@
+  image: "/project-stories/plymouth-university/plymouth-website.jpg"
+  image_description: "Plymouth website screenshot"
+
+plymouth_cio_video:
+  youtube_id: "nTSKD1PG1e4"
+  thumbnail: "/project-stories/plymouth-university/video-still.jpg"
+
+design_testing:
+  title: "​​Every design fully tested with students before build"
+  content: "Since the University and students had different ideas about what was required, the team ran sessions with all stakeholders but with particular emphasis on the main users (students). They then worked to deliver the most important features first, iterating to add further functionality on the basis of the evidence gathered for what would be most effective."
+  image: "/project-stories/plymouth-university/design-testing.jpg"
+  image_description: "Design sessions with Plymouth University students"
+
+benefits_quote:
+  quote: "Unboxed Consulting have created a far more attractive and appealing shop-window, which we are starting to see the benefits of."
+  source: "Paul Westmore, Interim CIO, Plymouth University"
+
+agile:
+  title: "Advanced Agile deployed for maximum value captured"
+  content: "In line with the most powerful elements of Agile philosophy, the new site was released almost immediately, alongside the old site. No clunky, delayed, high stress site changeover. New functionality was being released early and often, directly in line with user feedback. Our team co-located with Plymouth University, living in a specially rented flat in Plymouth."
+  image: "/project-stories/plymouth-university/design-sketch.jpg"
+  image_description: "One of the first sketches of Plymouth University website"
+
+statistics_1:
+  title: "Plymouth University in numbers"
+  number: "10,000+"
+  text: "webpages of content maintained"
+
+statistics_2:
+  number: "50+"
+  text: "Rank increase in the Sitemorse 2015 Q3/UK Universities and Higher Education list"
+
+team_value:
+  title: "Curation team given new efficiency tools, creating long term value"
+  content: "We worked with the team responsible for maintaining the website’s 10,000+ pages of content and adding new material. Kanban tools were taught to improve efficiency and give performance metrics, with the result that duration of the commissioning and uploading process dropped dramatically. The Ruby on Rails skills of the internal team so they can maintain the site efficiently going forward."
+  image: "/project-stories/plymouth-university/team-meeting.jpg"
+  image_description: "Meeting of Plymouth Website project team"
+
+team_quote:
+  quote: "Unboxed Consulting excelled in that they were able to put a team down in Plymouth to skill up our in-house team, something that many suppliers are unwilling to do."
+  source: "Paul Westmore, Plymouth University"
+site:
+  link: "https://www.plymouth.ac.uk/"
+  text: "Visit the site and check it out for yourself"
+
+team:
+  people:
```

Comment:
Fixed in https://github.com/unboxed/ubxd_web_refresh/commit/9563f01ab0f25c5a58d625b30c61961b1188af23